Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs

```diff
@@ -248,11 +248,10 @@ public class ZitiIdentity {
 				case ZitiEventFlags.ZitiServiceEvent:
 					Native.ziti_service_event ziti_service_event = Marshal.PtrToStructure<Native.ziti_service_event>(ziti_event_t);
 
-					ZitiServiceEvent serviceEvent = new ZitiServiceEvent() {
+					ZitiServiceEvent serviceEvent = new ZitiServiceEvent(ziti_context) {

```

Comment:
ziti_context is mandatory to create the ziti service objects. So moved the parameter to the constructor.